src/index.ts

```diff
@@ -9,6 +9,7 @@ export async function run(): Promise<void> {
     const registry = core.getInput('registry', { required: true });
     const username = core.getInput('username', { required: true });
     const password = core.getInput('password', { required: true });
+    const tlsVerify = core.getInput('tls-verify') || 'true';
```

Comment:
Are there any valid values for this other than 'true' and 'false' ? should we make it a boolean?

if the user inputs 'yes' or 'abc' what should happen?

`tlsVerify: boolean = core.getInput('tls-verify') === 'true'`

Reply:
forget it, just leave it as the string. it's less confusing that way.